agent/ecscni/netconfig_linux.go

```diff
@@ -176,6 +176,8 @@ func NewAppMeshConfig(appMesh *appmesh.AppMesh, cfg *Config) (string, *libcni.Ne
 // NewServiceConnectNetworkConfig creates a new ServiceConnect CNI network configuration
 func NewServiceConnectNetworkConfig(
 	scConfig *serviceconnect.Config,
+	redirectMode RedirectMode,
+	shouldIncludeRedirectIP bool,
```

Comment:
nit. `shouldIncludeRedirectIP, enableIPv4, enableIPv6 bool,`  there of them are all boolean.

Reply:
Will update to ist each in its own line with an explicit type.